### ISSUE TITLE
Part 3. Why constrain both max time and n?

### ISSUE THREAD

**Issue by JayRod12** [HPCE/hpce-2017-cw2#56]
I'm a little bit confused with what we're expected to do for part 3. 
The spec says to plot K=[1..16] vs time for n=[64,256,1024,4096] and it also says that the test runs should go up to more than 30 seconds. But as far as I understand, if we only plot the values of K vs time for the proposed values of n, then the time will a lot smaller than 30 seconds for the largest n value (i.e. for n=4096. For this n, I see an associated execution time of ~0.5 seconds when K=6 or so).

I might be missing the point, could someone clear things up?

Thanks!

**Comment by m8pple** [comment]
Increasing the time limit is mainly because some older machines and laptops will be significantly
slower, and may take more than the default of 10 seconds to get up to 4096, especially
when K=1. Those with mobile chips with small caches are particularly likely to see slow times.

So it is not saying it will _need_ 30 seconds, but more that it may need _up to_ 30 seconds.

I think any machine will be able to do it for n=4096 in <30 seconds with a reasonable
value of K; it is the K=1 case that is sometimes most problematic.

On faster machines (and on different compilers), it may take substantially less time, particularly for K>2.
In those circumstances you might want to include n=16384 as well, though that requires a time
limit more than 30 seconds on my desktop for K=1. I chose the numbers with ratios of 4 to try
to show how substantial the quadratic scaling is, and also to try to get big and small numbers
on the same graph.

I'll try to reword it to convey that idea more clearly.
